reactive/pom.xml

```diff
@@ -93,6 +94,12 @@
       <artifactId>mockwebserver</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Can we avoid the extra dependency?

Reply:
Could you tell me what your concern is, the apache common library is a widely used library? Can I just copy the related code here since it is under apache license too?

Reply:
Big dependency tree, conflicts with other libraries (which would lead to runtime errors)... When possible, we avoid extra dependencies on feign